app/src/main/java/org/mozilla/focus/utils/FileUtils.kt

```diff
@@ -10,16 +10,20 @@ import edu.umd.cs.findbugs.annotations.SuppressFBWarnings
 
 import java.io.File
 
+// WebView directory and contents.
 private const val WEBVIEW_DIRECTORY = "app_webview"
 private const val LOCAL_STORAGE_DIR = "Local Storage"
 
+// Cache directory contents.
+private const val WEBVIEW_CACHE_DIR = "org.chromium.android_webview"
+
 class FileUtils {
     companion object {
         @JvmStatic
-        fun truncateCacheDirectory(context: Context): Boolean {
-            val cacheDirectory = context.cacheDir
-            return cacheDirectory.exists() && deleteContent(cacheDirectory)
-        }
+        fun truncateCacheDirectory(context: Context) = deleteContent(context.cacheDir, doNotEraseWhitelist = setOf(
+                // If the folder or its contents are deleted, WebView will stop using the disk cache entirely.
+                WEBVIEW_CACHE_DIR
+        ))
```

Comment:
If we do not remove the webview cache files then I do not see any need to truncate the cache directory at all. The only reason why we did that is to get rid of the webview files. And just to be sure we remove everything in there to avoid guessing the right folder.
So I guess if we want to keep the files then let's remove this completely.

Reply:
My initial PR did this but, after my interpretation of your comments, I decided we should be more aggressive about deleting another other files that make their way in there: https://github.com/mozilla-mobile/focus-android/pull/1570#issuecomment-340611638

@pocmo Are you sure you want to remove the cache deletion entirely?